### ISSUE TITLE
Bug: TypeError exception when duplicating macros [Foundry 0.8.6]

### ISSUE THREAD

**Issue by ravngr** [earlSt1/vtt-macro-folders#20]
I just upgraded to the 0.8.x series and it looks like some under-the-hood API changes broke macro duplication. Attempting to duplicate a macro results in the following error in the debug console (in Firefox):

```
21:41:48.965 Uncaught (in promise) TypeError: entity.clone(...).then is not a function
    callback https://<hostname>/modules/macro-folders/macro-folders.js:575
    render https://<hostname>/scripts/foundry.js:20148
    jQuery 9
    render https://<hostname>/scripts/foundry.js:20145
    bind https://<hostname>/scripts/foundry.js:20088
    jQuery 2

macro-folders.js:575:69
```

Re-enabling and re-installing the module didn't result in any change.

**Comment by earlSt1** [comment]
Thanks for the report, ill check this out when i get the chance (currently waiting for internet in my new home)

**Comment by earlSt1** [comment]
Ok, internet and computer up and running. What system are you using out of curiosity?

**Comment by ravngr** [comment]
Which game system? DnD5e 1.3.3

**Comment by earlSt1** [comment]
Ok i think i've cracked it. Can you try uninstalling and installing using the manifest below:
https://raw.githubusercontent.com/earlSt1/vtt-macro-folders/08x-update/test/module.json

**Comment by ravngr** [comment]
Yep I think you've got it, that version works perfectly.

**Comment by earlSt1** [comment]
Fixed in v1.2.21
